src/Request.php

```diff
@@ -32,37 +33,32 @@ class Request extends Context
      */
     public function connect()
     {
-        if ($this->getProperty('ssl_options')) {
-            $this->connection = new AMQPSSLConnection(
-                $this->getProperty('host'),
-                $this->getProperty('port'),
-                $this->getProperty('username'),
-                $this->getProperty('password'),
-                $this->getProperty('vhost'),
-                $this->getProperty('ssl_options'),
-                $this->getProperty('connect_options')
-            );
-        } else {
-            $this->connection = new AMQPStreamConnection(
-                $this->getProperty('host'),
-                $this->getProperty('port'),
-                $this->getProperty('username'),
-                $this->getProperty('password'),
-                $this->getProperty('vhost'),
-                $this->getConnectOption('insist', false),
-                $this->getConnectOption('login_method', 'AMQPLAIN'),
-                $this->getConnectOption('login_response', null),
-                $this->getConnectOption('locale', 3),
-                $this->getConnectOption('connection_timeout', 3.0),
-                $this->getConnectOption('read_write_timeout', 130),
-                $this->getConnectOption('context', null),
-                $this->getConnectOption('keepalive', false),
-                $this->getConnectOption('heartbeat', 60),
-                $this->getConnectOption('channel_rpc_timeout', 0.0),
```

Comment:
Do we need to call setChannelRPCTimeout to set this value to be backward compatible?

Reply:
Good call, I must have missed this one